packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -406,7 +408,7 @@ export class GarbageCollector implements IGarbageCollector {
     private latestSummaryGCVersion: GCVersion;
 
     // Keeps track of the GC state from the last run.
-    private previousGCDataFromLastRun: IGarbageCollectionData | undefined;
```

Comment:
Hahaha, I remember liking `previousGCData`, and you like `gcDataFromLastRun`. Do we differentiate between `gcDataFromLastRun` and `currentGCData`?

Reply:
Lol yeah. I don't mind previousGCData either. It's just that the combination was redundant :-).